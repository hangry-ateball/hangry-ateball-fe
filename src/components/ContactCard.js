import React from 'react'
import { View, Text, TouchableOpacity } from 'react-native'
import { sendSMS } from '../apiCalls'

const ContactCard = ({ name, phoneNumber, digits, message, closeModal}) => {
  return (
    <TouchableOpacity onPress={() => {
      sendSMS(digits, '18647778548', message)
      closeModal()
    }}
    >
      <View style={styles.cardContainer}>
        <Text style={styles.contactName}>{name}</Text>
        <Text style={styles.contactNum}>{phoneNumber}</Text>
      </View>
    </TouchableOpacity>
  )
}

const styles = StyleSheet.create({
  cardContainer: { 
    minHeight: 70, 
    padding: 10, 
    borderBottomWidth: 1, 
    width: '100%', 
    backgroundColor: '#000065' 
  },
  contactName: {
    color: '#fff', 
    fontWeight: 'bold', 
    fontSize: 26
  },
  contactNum: {
    color: '#fff', 
    fontWeight: 'bold'
  }
});

export default ContactCard