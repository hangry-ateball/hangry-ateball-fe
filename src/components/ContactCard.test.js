import React from 'react';
import {shallow} from 'enzyme';
import ContactCard from './ContactCard';
describe('StartScreen', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
           const wrapper = shallow(<ContactCard/>);
            expect(wrapper).toMatchSnapshot();
        });
    });
});