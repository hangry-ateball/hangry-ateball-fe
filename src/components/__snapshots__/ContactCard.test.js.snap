// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartScreen Rendering should match to snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
>
  <View
    style={
      Object {
        "backgroundColor": "#000065",
        "borderBottomWidth": 1,
        "minHeight": 70,
        "padding": 10,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 26,
          "fontWeight": "bold",
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#fff",
          "fontWeight": "bold",
        }
      }
    />
  </View>
</TouchableOpacity>
`;
